src/utils/dictionaries/amm/mainnet.ts

```diff
@@ -12,7 +12,7 @@ export const mainnetAmm = [
     '0xe76Ea314b32fCf641C6c57f14110c5Baa1e45ff4',
     '0x09C5faF7723b13434ABdF1a65aB1B667BC02A902',
   )
-    .setLootDropColor(LootDropColors.Purple)
+    .setLootDropColor(PromotionColor.Purple)
```

Comment:
Rename the method to `setPromotionColor`.

Reply:
done